modules/geo-layers/src/mvt-layer/mvt-layer.js

```diff
@@ -267,5 +293,17 @@ function transformTileCoordsToWGS84(object, tile, viewport) {
   return feature;
 }
 
+function conversionToWGS84isRequired(tileCoords, isWGS84) {
+  return tileCoords === 'wgs84' && !isWGS84;
+}
+
+function updateTileWithWGS84Coords(tile, content, viewport) {
```

Comment:
You are not updating the tile, just transforming the content. Name the function accordingly.